src/mscorlib/src/System/Runtime/CompilerServices/TaskAwaiter.cs

```diff
@@ -557,6 +563,7 @@ public void UnsafeOnCompleted(Action continuation)
             /// <exception cref="System.NullReferenceException">The awaiter was not properly initialized.</exception>
             /// <exception cref="System.Threading.Tasks.TaskCanceledException">The task was canceled.</exception>
             /// <exception cref="System.Exception">The task completed in a Faulted state.</exception>
+            [StackTraceHidden]
```

Comment:
You'd want these on the corresponding methods in ValueTask's awaiters, too, right?
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/Runtime/CompilerServices/ValueTaskAwaiter.cs
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/Runtime/CompilerServices/ConfiguredValueTaskAwaitable.cs

Reply:
Added and moved Attribute to shared partition